docs/index.md

```diff
@@ -93,7 +93,7 @@ The example paragraph below has the following dynamic elements: a tooltip, a pop
 In <tooltip content="Computer Science">CS</tooltip>, a binary tree is a <trigger for="pop:index-tree">tree data structure</trigger> in which each node has at most two children, which are referred to as the _left child_ and the _right child_. <trigger trigger="click" for="modal:index-primitive">Primitive data types</trigger> on the other hand ...
 
 <popover id="pop:index-tree" header="An example tree data structure" placement="top">
-<div slot="content">
+<div #content>
```

Comment:
Seems like this popover is not rendered as well.

![image](https://user-images.githubusercontent.com/47470981/108535962-d8e08680-7316-11eb-95dc-f58f6738b7a6.png)